build/patch-charts.sh

```diff
@@ -254,12 +254,6 @@ function patchSidecarInjector() {
       - SETUID
   }' ${HELM_DIR}/istio/files/injection-template.yaml
 
-  # add annotation for Multus & Istio CNI
-  sed_wrap -i -e 's/^\(.*injectedAnnotations:.*\)$/\1\
-    \{\{- if .Values.istio_cni.enabled \}\}\
-      k8s.v1.cni.cncf.io\/networks: \{\{.Values.istio_cni.istio_cni_network\}\}\
-    \{\{- end \}\}/' ${HELM_DIR}/istio/templates/sidecar-injector-configmap.yaml
```

Comment:
I'm afraid we still need this, as the injection of `podRedirectAnnot` is disabled by default in 1.1. We should replace the check for `.Values.istio_cni.enabled` (as that'll always be true with Maistra) with a check for `not .Values.sidecarInjectorWebhook.injectPodRedirectAnnot`

cc @luksa

Reply:
I thought we had recently enabled this, I'll double check today and fix

Reply:
I've verified it's not enabled by default, apologies for this.  I'll rework the fix.

Reply:
@dgn Can you please take another look?

Reply:
LGTM, thanks